src/core/lib/gpr/fork.cc

```diff
@@ -32,14 +34,27 @@
  *       AROUND VERY SPECIFIC USE CASES.
  */
 
+typedef struct fork_state {
+  /* This variable starts counting at 2.  We use g_exec_ctx_count=1 to block
+   * new exec_ctx's from being created
+   */
+  gpr_atm exec_ctx_count;
+  gpr_mu mu;
+  gpr_cv thd_cv;
```

Comment:
Can you write a comment to differentiate these 2 CVs.

Reply:
Added a comment.